PersonsApi/Swagger/SwaggerOptions.cs

```diff
@@ -0,0 +1,63 @@
+
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace PersonsApi.Swagger
+{
+    public class SwaggerOptions : IConfigureNamedOptions<SwaggerGenOptions>
+    {
+        private readonly IApiVersionDescriptionProvider _provider;
+
+        private const string ApiKeyName = "ApiKey";
+        private const string ServiceTitle = "Persons API";
+        private const string ServiceDescription = "The Academies API provides users with access to a variety of data " +
+                                           "regarding academies and trusts in England.\n\n" +
+                                           "The available data includes general data acadamy transfers and " +
+                                           "applications to become an academy and is compiled from a variety of internal and " +
+                                           "external services.";
+        private const string ContactName = "Support";
+        private const string ContactEmail = "servicedelivery.rdd@education.gov.uk";
```

Comment:
Don't think this email is valid anymore, so should probably be updated, then again, it's likely the same in Academies API swagger so needs updating there too